packages/core-mobile/app/screens/watchlist/TokenDetails/TokenDetails.tsx

```diff
@@ -42,7 +43,19 @@ const CHART_THICKNESS = 4
 
 type ScreenProps = WalletScreenProps<typeof AppNavigation.Wallet.TokenDetail>
 
-const TokenDetail: FC = () => {
+const isValidNumber = (value: unknown): value is number => {
+  return typeof value === 'number' && !isNaN(value) && value > 0
+}
+
+// https://www.google.com/ -> google.com
+const stripUrl = (url: string): string => {
```

Comment:
```suggestion
const getDomainFromUrl = (url: string): string => {
```

and please move these functions to the util folder.